Demo/Common/IntegrationExamples/AdvancedFlowExample.swift

```diff
@@ -0,0 +1,374 @@
+////
+//// Copyright (c) 2023 Adyen N.V.
+////
+//// This file is open source and available under the MIT license. See the LICENSE file for more info.
+////
+//
+
+import AdyenActions
+import AdyenComponents
+import AdyenDropIn
+
+internal final class AdvancedFlowExample: AdvancedFlowExampleProtocol {
+
+    internal var advancedFlowComponent: PresentableComponent?
```

Comment:
I think this is the `currentComponent` from before. The old name was more clear as to its purpose I think. Suggest to keep it as before.

Reply:
Let's call it `dropInComponent` ?

Reply:
Made the change: da858bb9a858774fa139005a65bb215f94ed0c10